transports/azure-service-bus-netstandard/compatibility.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Compatibility
+summary: Describes the requirements for backward compatibility with ASB transport
+component: ASBS
+tags:
+ - Azure
+reviewed: 2018-06-19
+---
+
+## Backward compatibility
+
+The Azure Service Bus .Net Standard transport is backward compatible with the Azure Service Bus transport under a certain set of preconditions.
+
+### Preconditions
+
+#### Forwarding topology
+
+The Azure Service Bus .Net Standard transport only supports the forwarding topology, an entity layout where a shared set of topics is used for publishing between the endpoints.
```

Comment:
> a shared set of topics is used for publishing between the endpoints.

Single topic only. ASB has removed "bundling" and by default it's one topic only.